engine/src/main/java/com/datatorrent/stram/debug/StdOutErrLog.java

```diff
@@ -52,6 +54,7 @@ public static void tieSystemOutAndErrToLog()
       logger.warn("found appender {} instead of RollingFileAppender", appender);
     }
 
+    LoggerUtil.addAppenders();
```

Comment:
Added reinitialization of a file appender. Please see the update in LoggerUtil.java:

  private static void initializeFileAppender()
  {
    fileAppender = getFileAppender();
    shouldFetchLogFileInfo = shouldFetchLogFileInformation();
  }

  public static void addAppenders(String[] names, String args, String propertySeparator)
  {
    try {
      Properties properties = new Properties();
      properties.load(new StringReader(args.replaceAll(propertySeparator, "\n")));
      for (String name : names) {
        addAppender(name, properties);
      }
      logger.debug("reinitializing file appender");
      initializeFileAppender();

    } catch (IOException ex) {
      ;
    }
  }